test/utils.py

```diff
@@ -106,9 +130,11 @@ def __init__(self, cartridge_cmd):
 
         self._processes = []
         self._instance_pids = set()
+        self._subprocess = None
 
     def start(self, project, instances=[], daemonized=False, stateboard=False, stateboard_only=False,
-              cfg=None, script=None, run_dir=None, data_dir=None, log_dir=None):
+              cfg=None, script=None, run_dir=None, data_dir=None, log_dir=None,
+              capture_output=False, exp_rc=0):
```

Comment:
`exp_rc ` I understand that it means "return code"... But may be it's better to give directly "expected_return_code" name for this parameter.

Also seems this function accepts too many parameters. I can advice a reconsider it's structure for simplification.

Reply:
Maybe sometimes...